elastic/kibana
# [Response Ops] Replace scripted metric aggs in alerting telemetry queries with terms aggregations

Resolves https://github.com/elastic/kibana/issues/132672

## Summary

* Split the queries in `alerting_telemetry` into 3 files, one for each index the telemetry task queries
* Replaced scripted metric aggregations that collected information by rule type into terms aggregations with subaggregations.

## To Verify
* Update alerting telemetry to run more frequently
* Create rules with connectors and verify that the alerting telemetry task runs without errors and that the telemetry it collects looks correct.

### Checklist
